UPGRADE-1.0.md

```diff
@@ -1,5 +1,36 @@
 # Upgrade to 1.0
 
+## Angular 1.6
+Ng-admin is now ready for Angular 1.6 for security reasons and we strongly recommend you to migrate too.
```

Comment:
s/ for security reasons and we strongly recommend you to migrate too./ instead of 1.4, mainly for security reasons. We advise you to update your application too./